Components/images/image.css

```diff
@@ -3,7 +3,6 @@
     block-size: auto;
     font-size: inherit;
     display: block;
-    margin: var(--space-1);
     max-width: 100%;
```

Comment:
Just a suggestion: If you are using block size instead of height then you should also use max-inline-size in place of max-width.

Reply:
Thank you! 
Yes, I used it within the class definition above before the change.